modules/manage/pages/manage-xdcr/replicate-using-scopes-and-collections.adoc

```diff
@@ -262,6 +275,20 @@ Note the confirmatory notification that appears underneath the replication-defin
 As this indicates, if a replication is defined to include any destination-entity &#8212; bucket, scope, or collection &#8212; that does not exist, the entity will be ignored, and no attempt will be made to replicate data to it.
 However, if other specified entities are valid, replication to them will proceed.
 
+. Establish appropriate settings for _deletion filters_.
+These filters are individually described in xref:manage:manage-xdcr/filter-xdcr-replication.adoc#deletion-filters[Deletion Filters].
+Detailed information on their significance for migration is provided in xref:learn:clusters-and-availability/xdcr-filtering.adoc#using-deletion-filters[Using Deletion Filters].
++
+Left-click on the *Filter Replication* toggle:
++
+image::manage-xdcr/filter-replication-toggle.png[,180,align=left]
++
+When the *Filter Replication* panel opens, access the *Deletion Filters*:
++
+image::manage-xdcr/filter-xdcr-deletion-filters.png[,320,align=left]
++
+For each filter, to ensure that deletion is replicated, leave the setting at its default (with the checkbox unchecked); and to ensure that deletion is _not_ replicated, check the checkbox.
```

Comment:
or expirations

Reply:
(Don't see deletion filter in XDCR adv filtering page)

If a source document contains an expiry, and the document is meant to live without the associated expiry on the target bucket, check the "TTL" checkbox...
This will cause the document's expiry to be set to 0 prior to transmission to the target.
However, this also means that the target bucket's expiry setting, if exists, will take effect.

Reply:
I've included mention of expirations. As mentioned above, I've added an introduction that provides info on each deletion filter, including TTL.